package.json

```diff
@@ -44,7 +44,7 @@
     "start-image_viewer": "node devtools/image_viewer/server.js",
     "start": "npm run start-test_dashboard",
     "baseline": "node tasks/baseline.js",
-    "preversion": "npm-link-check && npm dedupe",
+    "preversion": "npm-link-check && npm dedupe && npm ls --prod",
```

Comment:
cc @alexcjohnson @dfcreative 

This commit partly reverts https://github.com/plotly/plotly.js/commit/1d3a3f0ead64168f1764ad7fe215936964bd116d

`npm ls --prod` will fail only if a bundle dependency (i.e. ignoring dev-dependencies) is (1) missing, (2) needs to be updated or (3) is extraneous. This step ensures that we're not bundling `dist/` bundles with incorrect dependencies.